atomicapp/external/templates/nulecule/Dockerfile.tpl

```diff
@@ -0,0 +1,10 @@
+FROM projectatomic/atomicapp:0.4.1
+
+MAINTAINER Your Name <you@email.com>
+
+LABEL io.projectatomic.nulecule.specversion="0.0.2" \
+      io.projectatomic.nulecule.providers="kubernetes, docker" \
+      Build="docker build --rm --tag test/$app_name-atomicapp ."
```

Comment:
@cdrage.. its not common but I have seen it before. @rtnpro I say let's not include it as it may confuse the user more than help.